_includes/sqlstyle.guide.md

```diff
@@ -230,31 +196,68 @@ UPDATE albums
 ```sql
 SELECT a.title,
        a.release_date, a.recording_date, a.production_date -- grouped dates together
-  FROM albums AS a
- WHERE a.title = 'Charcoal Lane'
-    OR a.title = 'The New Danger';
+FROM albums AS a
+WHERE a.title = 'Charcoal Lane'
+   OR a.title = 'The New Danger';
 ```
 
 ### Indentation
 
 To ensure that SQL is readable it is important that standards of indentation
 are followed.
 
+**ONLY** the fundamental keywords - `SELECT`, `FROM`, `WHERE`, `GROUP BY`, `HAVING`, `LIMIT`,
+and `ORDER BY`should be fully left justified.  Other clauses should be indented to the end of
+that keyword.
+
+```sql
+SELECT first_name,
+       last_name,
+       is_still_tippin_on_four_fours,
+       is_still_wrapped_in_four_vogues
+FROM rappers
+WHERE first_name = 'Mike'
+  AND last_name = 'Jones'
+```
+
+This allows the reader to quickly scan for the important building blocks of the query.
+
 #### Joins
 
-Joins should be indented to the other side of the river and grouped with a new
-line where necessary.
+Joins should be indented 2 spaces right from the `FROM` keyword
+
+Single line `JOIN`s are fine for simple situations
+
+```sql
+SELECT r.last_name
+FROM riders AS r
+  INNER JOIN bikes b ON r.bike_vin_num = b.vin_num
+  INNER JOIN crew c ON r.crew_chief_last_name = c.last_name
```

Comment:
I find left justified JOINs to be really problematic when reading because they aren't top level clauses.  I see lots of people left justifiying `JOIN` (and `ON`) but I really think that should stop.  JOIN and ON are not the same as FROM in terms of importance; they are clearly subclauses of the FROM block.